src/Sylius/Behat/Page/Admin/Crud/IndexPage.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page\Admin\Crud;
+
+use Behat\Mink\Session;
+use Sylius\Behat\Page\SymfonyPage;
+use Sylius\Behat\TableManipulatorInterface;
+use Symfony\Component\Routing\RouterInterface;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class IndexPage extends SymfonyPage implements IndexPageInterface
+{
+    /**
+     * @var string
+     */
+    protected $resourceName;
+
+    /**
+     * @var TableManipulatorInterface
+     */
+    protected $tableManipulator;
+
+    /**
+     * @var array
+     */
+    protected $elements = [
+        'message' => '.message',
+        'messageContent' => '.message > .content',
+        'table' => '.table',
+    ];
+
+    /**
+     * @param Session $session
+     * @param array $parameters
+     * @param RouterInterface $router
+     * @param TableManipulatorInterface $tableManipulator
+     * @param string $resourceName
+     */
+    public function __construct(
+        Session $session,
+        array $parameters,
+        RouterInterface $router,
+        TableManipulatorInterface $tableManipulator,
+        $resourceName
+    ) {
+        parent::__construct($session, $parameters, $router);
+
+        $this->tableManipulator = $tableManipulator;
+        $this->resourceName = $resourceName;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function hasSuccessMessage()
+    {
+        return $this->getElement('message')->hasClass('positive');
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isSuccessfullyCreated()
+    {
+        return $this->hasMessage(sprintf('Success %s has been successfully created.', ucfirst($this->resourceName)));
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isSuccessfullyUpdated()
+    {
+        return $this->hasMessage(sprintf('Success %s has been successfully updated.', ucfirst($this->resourceName)));
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isSuccessfullyDeleted()
+    {
+        return $this->hasMessage(sprintf('Success %s has been successfully deleted.', ucfirst($this->resourceName)));
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function isResourceOnPage(array $parameters)
+    {
+        $rows = $this->tableManipulator->getRowsWithFields($this->getElement('table'), $parameters);
+
+        return !empty($rows);
```

Comment:
> Returns FALSE if var exists and has a non-empty, non-zero value. Otherwise returns TRUE.
> 
> The following things are considered to be empty:
> 
> "" (an empty string)
> 0 (0 as an integer)
> 0.0 (0 as a float)
> "0" (0 as a string)
> NULL
> FALSE
> array() (an empty array)
> $var; (a variable declared, but without a value)

It just feels more safe :tada: